update-translations.sh

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env bash
+
+CWD=`pwd`
+DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+cd $DIRECTORY
+
+function log() {
+  if [[ -n $1 ]];then
+    printf "\033[0;36m\n$1\n\033[0m"
+  fi
+}
+
+function quit() {
+  if [[ -n $1 ]];then
+    printf "\033[0;31m\n\n$1\n\n\033[0m"
+  fi
+  cd $CWD
+  exit
+}
+
+function repo_checkout_master() {
+  if [[ -n $1 ]];then
+    log "checking out master for '$1'"
+    cd $DIRECTORY
+    cd "workspace-zen/$1/"
+    DIRTY=`git status --porcelain`
+
+    if [[ -n $DIRTY ]]; then
+      quit "$1 has unstaged / uncommitted changes, please fix and try again"
+    fi
+  fi
+}
+
+# Update the translations repo if we are able
+repo_checkout_master 'cp-translations'
+
+TRANSLATIONS_VERSION=$(cat package.json \
+  | grep version \
+  | head -1 \
+  | awk -F: '{ print $2 }' \
+  | sed 's/[ ",^]//g')
+
```

Comment:
Is `jq` available..? I mean, you've written this now, so maybe  it doesn't matter :)

Reply:
Alternatively, you could use `npx project-version` to get the version.

Reply:
Yes, okay - so found out I can do `jq '.dependencies."cp-translations"'` and `.dependencies."cp-translations" =  "$TRANSLATIONS_VERSION"` - will check that out, but means adding package.json here.